src/tests/unit/DocumentSelectionHandler.test.ts

```diff
@@ -0,0 +1,55 @@
+import { lambdaHandler } from "../../DocumentSelectionHandler";
+import { mock } from "jest-mock-extended";
+import { RESOURCE_NOT_FOUND, UNSUPPORTED_METHOD, VALID_REQUEST, INVALID_SESSION_ID, MISSING_SESSION_ID } from "./data/documentSelection-events";
+
+import { Response } from "../../utils/Response";
+import { HttpCodesEnum } from "../../utils/HttpCodesEnum";
+import { DocumentSelectionRequestProcessor } from "../../services/DocumentSelectionRequestProcessor";
+
+const mockDocumentSelectionRequestProcessor = mock<DocumentSelectionRequestProcessor>();
+jest.mock("../../utils/Config", () => {
+	return {
+		getParameter: jest.fn(() => {return "YOTIPRIVATEKEY";}),
+	};
+});
+
+jest.mock("../../services/AuthorizationRequestProcessor", () => {
```

Comment:
this should be DocumentSelectionRequestProcessor or I guess we dont need this mock here because at line 9, we are mocking the processor?

Reply:
Deleted